editor/components/icon-button/index.js

```diff
@@ -10,12 +10,13 @@ import './style.scss';
 import Button from '../button';
 import Dashicon from '../dashicon';
 
-function IconButton( { icon, label, className, ...additionalProps } ) {
+function IconButton( { icon, text, label, className, ...additionalProps } ) {
 	const classes = classnames( 'editor-icon-button', className );
 
 	return (
 		<Button { ...additionalProps } aria-label={ label } className={ classes }>
 			<Dashicon icon={ icon } />
+			{ text ? <span>{ text }</span> : null }
```

Comment:
```
function IconButton( { icon, children, label, className, ...additionalProps } ) {
  // ...
  { children ? <span>{ children }</span> : null } // This replaces the line 19
  // ..
}
```

And now instead of writing `<IconButton text="My Text" />`, you'd write `<IconButton>My Text</IconButton>`.